graphql/concrete_notification/dummy_data.ts

```diff
@@ -0,0 +1,20 @@
+import { ConcreteNotification, ConcreteNotificationState } from '../../common/entity/ConcreteNotification';
+
+const ids = Array.from({ length: 10 }, (_, i) => i + 1);
+const dummyMinutes = [0, 5, 10, 15, 20, 30, 60, 120, 150, 60 * 24, 60 * 48];
+
+// Dummy dates indexed by dummy ids
+export const getDummyCreatedAt = (id: number): Date => new Date(Date.now() - dummyMinutes[id] * 1000 * 60);
```

Comment:
What is the purpose of "createdAt"? Concrete Notifications have a "sentAt" field, which for SENT notifications is the date when the notification was sent out

Reply:
I was not sure about sentAt - is this the timestamp, when notification was created or when it was actually sent by mailJet? if sentAt is only set once when created and not connected to mailJet, I would use it instead

Reply:
c.f. https://github.com/corona-school/backend/blob/master/common/entity/ConcreteNotification.ts#L35

Reply:
As the name suggests, it's when the notification was sent ...

Reply:
okay, that means we need a new field createdAt to know when the notification was created, right? it seems prisma could populate it automatically for us on creation

Reply:
No? Why would the creation date of the database entry be of relevance?